wpeverest/everest-forms
# Fix - Post Meta Smart tag not working.

### All Submissions:


### Changes proposed in this Pull Request:
Previously, Post Smart meta smart tag was not not working. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Create a form.
2.  Add a post meta smart tag and verify if  the post meta smart tag working or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Post Meta Smart tag not working.
